### PR TITLE
sig: rbs:subtract is no longer needed 

### PR DESCRIPTION
**Pull request by tk0miya**
Now we completed the migration to RBS::Inline from "rbs prototype" and
handwritten types.  It's time to make RBS files under `sig/generated/`
primary.

In "rbs prototype" era, our types were conflicted.  But, after the
migration, they are not conflicted.

Therefore this stops subtracting types from each other.

Note: This contains https://github.com/kaigionrails/conference-app/pull/362.

### CODE REVIEW

File: lib/tasks/rbs.rake [hunk 1]
```diff
@@ -2,7 +2,7 @@ if Rails.env.local?
   require "rbs_rails/rake_task"
 
   namespace :rbs do
-    task setup: %i[clean collection generate load_routes rbs_rails:all subtract]
+    task setup: %i[clean collection generate load_routes rbs_rails:all validate]
```
Comment (tk0miya) [inline lib/tasks/rbs.rake:5]: It's better to run `rbs:validate` at the end of `rbs:setup` to validate all types.

File: sig/generated/channels/application_cable/channel.rbs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# Generated from app/channels/application_cable/channel.rb with RBS::Inline
```
Comment (tk0miya) [inline sig/generated/channels/application_cable/channel.rbs:1]: So far this comment headers from RBS::Inline were removed on subtracting. But it is kept generated from now on.

File: sig/handwritten/app/models/user.rbs [hunk 3]
```diff
@@ -8,7 +8,9 @@ class User < ApplicationRecord
 
   # User has profile and authentication_provider_github at least one
   def profile: () -> Profile
+             | ...
```
Comment (tk0miya) [inline sig/handwritten/app/models/user.rbs:11]: I added an overload statement (`| ...`) to avoid the conflicts with types from rbs_rails.

**Review by unasuke — APPROVED** [review]
🙆

**Pull request merged**